root/static/scripts/relationship-editor/components/DialogLinkType.js

```diff
@@ -291,6 +298,35 @@ const DialogLinkType = (React.memo<PropsT>(({
         <div aria-atomic="true" className="error" role="alert">
           {error}
         </div>
+        {isHelpVisible ? (
+          <div className="ar-descr">
+            {linkType === undefined ? (
+              <>
+                {exp.l(
+                  `{url|Documentation} about
+                     selectable relationship types.`,
```

Comment:
Hmm. This is technically not always true (for example, if you open a recording series, and try to link it to a RG, we will not show "part of" as selectable, but it will obviously still be listed in the relationships link).

Also, showing "Documentation about selectable relationship types" as a help sentence seems a bit weird - if we keep something like this I might go with "See the {url|Documentation about selectable/possible/available relationship types}"

I'm actually thinking that, given this is shown if a type is not selected yet, maybe the help should be something like this?:
"Select the type of relationship between {edited_entity} and the target ({url|$type-$type relationship list}}"

But obviously not a blocking issue, so we could merge like this for now.

Reply:
That’s fair. I rephrased it as a gentle sentence, avoided using the part about “target” which is in code only, and linked using the same “(more documentation)” title as when a relationship type is selected; See [change](https://github.com/metabrainz/musicbrainz-server/compare/3e98142193e267e0fa893450fbe55592fe3579b0..c6b17ab4f9fd7961f7e3d106ea816330c80ec02d).